### PR TITLE
vendor: use faster version instead compress/gzip

### PR DESCRIPTION
**Pull request by giuseppe**
use "github.com/klauspost/pgzip" instead of "compress/gzip".

some numbers:

```
## I am using a directory repository so to not depend on the network

# skopeo copy docker://tensorflow/tensorflow:latest dir:/var/tmp/tensorflow
# skopeo copy docker://fedora:latest dir:/var/tmp/fedora

## Each test is done with a fresh storage

## Before

# /usr/bin/time -v /usr/bin/skopeo copy dir:/var/tmp/tensorflow containers-storage:tensorflow
User time (seconds): 39.40
System time (seconds): 6.83
Percent of CPU this job got: 121%
Elapsed (wall clock) time (h:mm:ss or m:ss): 0:38.07
# /usr/bin/time -v /usr/bin/skopeo copy dir:/var/tmp/fedora containers-storage:fedora
User time (seconds): 8.32
System time (seconds): 1.62
Percent of CPU this job got: 128%
Elapsed (wall clock) time (h:mm:ss or m:ss): 0:07.72

## After

# /usr/bin/time -v ./skopeo copy dir:/var/tmp/tensorflow containers-storage:tensorflow
User time (seconds): 42.68
System time (seconds): 6.64
Percent of CPU this job got: 162%
Elapsed (wall clock) time (h:mm:ss or m:ss): 0:30.44
# /usr/bin/time -v ./skopeo copy dir:/var/tmp/tensorflow containers-storage:fedora
User time (seconds): 8.94
System time (seconds): 1.51
Percent of CPU this job got: 178%
Elapsed (wall clock) time (h:mm:ss or m:ss): 0:05.85
```

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -48,7 +48,7 @@ github.com/xeipuuv/gojsonschema master
 github.com/xeipuuv/gojsonreference master
 github.com/xeipuuv/gojsonpointer master
 go4.org master https://github.com/camlistore/go4
-github.com/ostreedev/ostree-go aeb02c6b6aa2889db3ef62f7855650755befd460
+github.com/ostreedev/ostree-go 56f3a639dbc0f2f5051c6d52dade28a882ba78ce
```
Comment (mtrmac) [inline vendor.conf]: Shouldn’t this be a separate PR? Or is it a necessary component?

Comment (giuseppe) [reply vendor.conf]: I can split it to a separate PR, but it is just to silence a compiler warning

**Comment by giuseppe** [comment]
re-vendored with `containers/image` and `containers/storage`

**Comment by rhatdan** [comment]
LGTM
@runcom @vrothberg Lets get this merged.

**Comment by vrothberg** [comment]
LGTM
@rhatdan, we need your Jedi power for merging.
